adi/rx_tx.py

```diff
@@ -136,7 +136,7 @@ def rx_enabled_channels(self, value: Union[List[int], List[str]]):
         translated to channel numbers.
         """
         if not value:
-            raise Exception("rx_enabled_channels cannot be empty")
+            return
```

Comment:
This is just an idea, but should we consider still displaying a warning here, without raising an exception? Something along the lines of `rx_enabled_channels empty; only debug interface is available`?

Reply:
yeah, I just re-implemented this with attribute class inheritance (no rx stuff) as per @tfcollins suggestion